echopype/mask/api.py

```diff
@@ -678,23 +677,85 @@ def detect_seafloor(
     params: Dict,
 ) -> xr.DataArray:
     """
-    Detect seafloor using the selected method and return a bottom line.
+    Dispatch seafloor detection to a chosen method and return a 1-D bottom line.
+
+    This function forwards ``ds`` and ``params`` to the selected implementation
+    (e.g., ``"basic"``, ``"blackwell"``). Any optional arguments omitted in
+    ``params`` are filled by that method's defaults.
 
     Parameters
     ----------
     ds : xr.Dataset
-        Sv dataset including ping_time and depth.
+        Dataset containing calibrated Sv and required coordinates (at minimum
+        ``ping_time`` and a vertical coordinate such as ``depth``), plus any
+        extra variables required by the chosen method.
     method : str
-        Name of the detection method to use (e.g., "basic", "blackwell").
+        Name of the detection method to use. Supported:
+          - ``"basic"``  → threshold-only detector
+          - ``"blackwell"`` → Sv + split-beam angle detector
     params : dict
-        Parameters for the detection function (method-specific).
+        Method-specific keyword arguments (see below). Unspecified keys use
+        the method's defaults.
+
+    Method-specific arguments
+    -------------------------
+    basic
+        var_name : str
+            Name of Sv variable (dB), e.g. ``"Sv"``.
+        channel : str
+            Channel identifier to select (e.g., ``"GPT  38 kHz ..."``).
+        threshold : float, default -50.0
+            Sv threshold in dB (lower bound). If the implementation also
+            supports an upper bound, a scalar is interpreted as a lower bound
+            only.
```

Comment:
Seeing what's under the blackwell method, I grabbed what you have in bottom_basic and modified it slightly below. Please check if it is correct.
```suggestion
        threshold : float, default -50.0
            Sv threshold(s) in dB. If a single float is given, it is treated as the
            lower bound and the upper bound is set to be 10 dB above the 
            lower bound. If a 2-tuple `(tmin, tmax)` is provided, both the lower
            and upper bounds are used directly.
```